components/style/gecko_conversions.rs

```diff
@@ -0,0 +1,32 @@
+//! This module contains conversion helpers between Servo and Gecko types
+//! Ideally, it would be in geckolib itself, but coherence
+//! forces us to keep the traits and implementations here
```

Comment:
I *think* it is because you can only implement them either where `CalcLengthOrPercentage` is defined or where `nsStyleCoord_CalcValue` is defined, and doing it in `gecko_bindings` is impossible because is in the other way of the dependency chain.

Other solution would be to implement a custom trait as `GeckoStyleCoordConvertible` is. I'm fine with any of both options, either using the standard `From` and paying this price, or using a custom type :)